### DISCUSSION TITLE
blog/2019-02-10-bayesian-robust-correlations-with-brms-and-why-you-should-love-student-s-t/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#27] [Comments]
# blog/2019-02-10-bayesian-robust-correlations-with-brms-and-why-you-should-love-student-s-t/

In this post, we'll show how Student's *t*-distribution can produce better correlation estimates when your data have outliers. As is often the case, we'll do so as Bayesians.

https://solomonkurz.netlify.app/blog/2019-02-10-bayesian-robust-correlations-with-brms-and-why-you-should-love-student-s-t/

**Comment by stanleyrazor** [comment]
Thanks for the excellent post! I have a question about the following:

Suppose we fit an intercept-only multivariate model with two responses, `y1` and `y2`, and specify the `t`-distribution as the likelihood (as above). We then use `stats::vcov` to obtain the variance-covariance matrix of the model parameters (just the intercepts in this case), and later apply `cov2cor` to convert this matrix into a correlation matrix (of the estimated intercepts).

Given this, I expected that the correlation between the intercepts of `y1` and `y2` (from `vcov` and `cov2cor`) should also give a robust estimate of the correlations between `y1` and `y2` (just like `rescor`)

My thinking is from this direction: When fitting an intercept-only model `y∼1`, the estimate of the intercept is essentially the mean of `y`. So, in the case of fitting an intercept-only `t`-distribution model, the estimated intercept is a robust estimate of the mean of `y`. Therefore, I would expect that the correlations between these robust means (t-models: `y1 ~ 1` & `y2 ~ 1`) should also be robust.

However, I have noticed that this is not the case. Can you help clarify about this and anything else I should know about multivariate models in this sense ?

**Comment by ASKurz** [comment]
Hey @stanleyrazor, thanks for your interest and your kind words. 

It took me a moment to catch what you were saying, but I think I follow now. I’m not so sure I’d expect the covariance among the standard errors of the intercepts to mirror the covariance between the variables in the data. However, I don’t think I have a deep enough background on the topic to make strong claims. We might need a statistician to chime in.
